### PR TITLE
reduce to one intro for doc chapter

### PR DESCRIPTION
**Pull request by SaranjeetKaur**

### CODE REVIEW

File: 07-documenting_R.Rmd [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
+This section summarises the [recommended style for writing the R help files](https://developer.r-project.org/Rds.html). The help files for R functions are written in 'R documentation' (.Rd) file format. It is a simple markup language with close resemblance to LaTeX. The .Rd file format can be further processed into a variety of formats, including LaTeX, HTML, and plain text. An example of the .Rd format is available for [mean.Rd](https://svn.r-project.org/R/trunk/src/library/base/man/mean.Rd). These are intended for writing the core documentations but may also be useful for package writers. Extensive details of writing R documentation files can be found in the [Writing R Extensions](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files) manual.
 
 Note: Many R package authors use the R package [roxygen2](https://cran.r-project.org/web/packages/roxygen2/index.html) to write documentation (which eventually creates the .Rd files), however, in this guide we focus on .Rd files only because the manual pages for the base distribution are written directly as .Rd files.
```
Comment (hturner) [inline 07-documenting_R.Rmd:13]:
```suggestion
The help files for R functions are written in 'R documentation' (`.Rd`) file format. It is a simple markup language with close resemblance to LaTeX. The `.Rd` file format can be further processed into a variety of formats, including LaTeX, HTML, and plain text. The `.Rd` files can be found in the `man` directory of the sources for the corresponding base package, for example <https://svn.r-project.org/R/trunk/src/library/base/man/mean.Rd>. Many authors of contributed R packages use the [roxygen2](https://cran.r-project.org/package=roxygen2) package to write documentation, which generates `.Rd` files from comments in the package `.R` files. However, the help files in the base distribution are maintained by directly editing the `.Rd` files.

This section gives an overview of the `.Rd` format and summarises the [guidelines for writing the R help files](https://developer.r-project.org/Rds.html). The guidelines are intended for contributors to base R, but may also be useful for authors of contributed packages. Full documentation on writing R documentation files can be found in the [Writing R Extensions](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files) manual.
```

Comment (hturner) [reply 07-documenting_R.Rmd:13]: When making this change, check the remaining chapter and update ".Rd" with "`.Rd`" where necessary (so it is formatted as code, as in R-exts).

**Review by hturner — CHANGES_REQUESTED** [review]
I've suggested some changes based on your edits. I think this affects https://github.com/r-devel/rdevguide/pull/148 too, so maybe update that PR before I review that one?

**Review by bettinagruen — COMMENTED** [review]
I agree with Heather's suggestions.

**Pull request merged**